src/vg.cpp

```diff
@@ -1842,6 +1846,15 @@ void VG::from_alleles(const map<long, set<vcflib::VariantAllele> >& altp,
 
             }
 
+            if (alt_node != NULL) {
+                auto nameIter = alleleNames.find(allele);
+                if (nameIter != alleleNames.end()) {
+                    for (auto& name : nameIter->second) {
+                        paths.append_mapping(name, alt_node->id());
```

Comment:
That sounds right... I thought we had tested things this far but maybe I was confused.

I didn't merge this because it's not complete so I'd expect there to be problems like this.